hammer-time.js

```diff
@@ -61,37 +61,43 @@ window.Hammer.time = {
 		return true;
 	},
 	shouldHammer: function( e ) {
-		var parentAction = this.hasParent( e.target );
+		var parentAction = e.target.hasParent;
```

Comment:
a hasParent is stored on target element like lastStart.
if hasParent should guard, lastStart should also guard.

may be I don't understand , I think lastStart and hasParent are same case.